cli/src/main.rs

```diff
@@ -46,12 +55,21 @@ enum TapSignerCommand {
     Certs,
     /// Read the pubkey (requires CVC)
     Read,
+    /// This command is only used once to initialize the card.
+    New,
 }
 
 fn main() -> Result<(), Error> {
     // figure out what type of card we have before parsing cli args
+    #[cfg(not(feature = "emulator"))]
     let mut card = pcsc::find_first()?;
 
+    // if emulator feature enabled override pcsc card
+    #[cfg(feature = "emulator")]
+    let mut card = emulator::find_emulator()?;
```

Comment:
To your knowledge, is it possbile to consolidate the card definition? I tried this:
```
let mut card = if cfg!(feature = "emulator") {
        emulator::find_emulator()?
    } else {
        pcsc::find_first()?
    };
``` 
but the compiler doesn't recognize the features inside the brackets